### DISCUSSION TITLE
Executing 'YARN' in my project failed （license: Command failed.）

### DISCUSSION THREAD

**Discussion by co-index** [eclipse-theia/theia#11697] [Q&A] [answered]
The host machine：macOS M1 
The virtual machine：Ubuntu 20.04.5(arm64)

  This project works correctly on both macOS(M1) and Ubuntu(amd64). But when I want this project to run in Ubuntu(arm64), some errors were thrown in the terminal. 
  I have searched for all possibilities in Chinese search engines and Google, but have not found similar problems(I'm a rookie). I hope to get your help.

next is error detail: 
![截屏2022-09-25 19 58 12](https://user-images.githubusercontent.com/89052431/192142252-54e45243-9891-432c-b356-0c23b5c9602c.png)

Please leave a message if you need more details. Best wishes to you  :)

**Comment by msujew** [comment, marked as answer]
Hey @MooooooonLight,

It seems to me like `license` isn't one of Theia's dependencies but is rather a module in your monorepo. Note the path `neusar_ide_monorepo/packages/license`. Is that the case? If so, this is unrelated to Theia and you need to fix it yourself in order to build correctly on Ubuntu+ARM64.

See also:

![image](https://user-images.githubusercontent.com/4377073/192155135-38cb9949-6e62-4045-b774-6c1e5447eb85.png)

**Comment by co-index** [comment]
Hey@msujew 

Thank you for your answer, and according to this, i found out that it really wasn't Theia's problem. I go to the lisence directory and execute “yarn build”, some error in here. I will analyze the reasons for the failure accordingly. 

Best wishes for you:)
